stytch/consumer/models/sessions.py

```diff
@@ -301,12 +304,11 @@ class TwitterOAuthFactor(pydantic.BaseModel):
     email_id: str
     provider_subject: str
 
-# MANUAL(WebAuthnFactor)(Types)
+
 class WebAuthnFactor(pydantic.BaseModel):
     webauthn_registration_id: str
     domain: str
-    user_agent: Optional[str] = None
-# ENDMANUAL(WebAuthnFactor)
+    user_agent: str
```

Comment:
This was already defined above, so now there are two. Either it *shouldn't* be manual (if it's already defined in the proto defs) or you just need to rerun once more so that it gets deduped

Reply:
Yeah the manual was removed in another PR and is fixed now when I merged main

Reply:
This was done due to this ticket: https://linear.app/stytch/issue/OBACK-63/[python-sdk]-webauthn-factor-user-agent-field-required-error. We're ok rolling this back?

Reply:
Good catch. This would *not* be okay to roll back. Instead we need to mark this as an explicitly optional field in the protos, cc @taronish-stytch

Reply:
https://github.com/stytchauth/protobuf/pull/956

Reply:
https://github.com/stytchauth/stytch-python/pull/185/commits/b188b7c24cd1d642104ca8f6f91f177c24503444